lib/Config.ts

```diff
@@ -150,4 +150,36 @@ class Config {
   }
 }
 
+export const convertConfigToToml = () => {
+  let result = '';
+
+  const recursivelyConvertJsonToToml = (json: any, prefix: string) => {
+    const nestedPairs: any = [];
+    const simplePairs: any = [];
+
+    if (!json) return;
+
+    Object.keys(json).sort().forEach((key: any) => {
+      if (Object.prototype.toString.call(json[key]) !== '[object Function]') {
+        (isPlainObject(json[key]) ? nestedPairs : simplePairs).push([key, json[key]]);
+      }
+    });
+
+    if (!(prefix === '' || simplePairs.length === 0)) {
+      result += '[' + prefix + ']\n';
+    }
+
+    simplePairs.forEach((value: any) => {
+      result += value[0] + ' = ' + value[1] + '\n';
+    });
+    nestedPairs.forEach((value: any) => {
+      recursivelyConvertJsonToToml(value[1], (prefix === null || prefix === '') ? value[0] : [prefix, value[0]].join('.'));
+    });
+  };
+
+  recursivelyConvertJsonToToml(new Config(), '');
+
+  fs.writeFileSync('../xud.conf', result);
```

Comment:
Do we want to add this file to our source repo? if not, let's add it to `.gitignore`

Reply:
Yeah I'm thinking it would make sense not to have it in source given we have this script. It'd be a bit of a hassle to update the file in source each time the default config changes, and if anyone wants this for themselves they can just run the script.

Reply:
Elsewhere we're using `path.join` and `__dirname` for getting relative paths so let's stick to that if possible, like `path.join(__dirname, '..', 'xud.conf');`

Reply:
Sorry for the confusion over keeping the file in the repo per #335, I wasn't considering the ease with which someone who wanted it could generate it for themselves.